src/sidebar/markdown-commands.js

```diff
@@ -209,19 +209,20 @@ function endOfLine(str, pos) {
  * @param {EditorState} state - The initial state of the input field
  * @param {number} start - The start position within the input text
  * @param {number} end - The end position within the input text
- * @param {(s: EditorState, start: number, end: number) => EditorState} callback
+ * @param {(s: EditorState, start: number, end: number, lineNum: number) => EditorState} callback
```

Comment:
As a naming convention I would suggest to use "number" in names to refer to 1-based counters and "index" for 0-based counters. This will avoid confusion with other places in the app where we have names like `pageNumber` which refer to 1-based page numbers in PDF.js. An explicit "index" suffix is only needed in cases where it is potentially ambiguous (eg. no need for an "index" suffix for `start` or `end` here).

Reply:
Makes sense.